src/python/WMCore/WMSpec/StdSpecs/StdBase.py

```diff
@@ -937,18 +937,16 @@ def getWorkloadArguments():
                                          "attr": "priority"},
                      "VoGroup": {"default": "unknown", "attr": "owner_vogroup"},
                      "VoRole": {"default": "unknown", "attr": "owner_vorole"},
-                     "Campaign": {"optional": False},
-                     "AcquisitionEra": {"default": "FAKE", "validate": acqname,
-                                        "assign_optional": False},
+                     "Campaign": {"optional": True}, # default value will be AcquisitionEra
```

Comment:
Technically default value is not None but AcquisitionEra. 
I thought about it for the TaskChain case with AcquisitionEra different for each task, I think in that case it shouldn't set the default value to "" or None. I am not sure what would be the intention on that but I thought we shouldn't assume anything for that. So I thought it would be safer either they specify Campaign other wise not allow create the requests